weave

```diff
@@ -979,6 +1007,7 @@ case "$COMMAND" in
         shift $CIDR_COUNT
         [ $# -eq 1 ] || usage
         CONTAINER=$(container_id $1)
+        ipam_cidrs $CONTAINER $CIDR_ARGS
```

Comment:
huh? We allocate IP addresses when *detaching* a container? Surely the semantics for subnets here should be "remove (and free in IPAM) and IP address in the given subnet that belongs to the container".

It seems to me that there is actually a bug / missing feature on default here; surely a) detaching an allocated IP should free it, and b) zero-arg detach should detach any allocation from the default ipam subnet.